config/samples/cluster_v1_cri.yaml

```diff
@@ -0,0 +1,387 @@
+      # transport:
+      #   tlsSecret:
+      #     name: ytsaurus-rpc-wildcard
+      #   tlsRequired: true
+      #   tlsPeerAlternativeHostName: ...
+
+
+  dataNodes:
+    - instanceCount: 1
+
+      loggers: *loggers
+
+      locations:
+        - locationType: Logs
+          path: /yt/data-node-logs
+        - locationType: ChunkStore
+          path: /yt/node-data/chunk-store
+
+      volumeMounts:
+        - name: data-node-logs
+          mountPath: /yt/data-node-logs
+        - name: node-data
+          mountPath: /yt/node-data
+
+      volumeClaimTemplates:
+        - metadata:
+            name: node-data
+          spec:
+            accessModes: [ "ReadWriteOnce" ]
+            resources:
+              requests:
+                storage: 20Gi
+        - metadata:
+            name: data-node-logs
+          spec: *logsVolumeSpec
+
+
+  execNodes:
+    - instanceCount: 1
+
+      loggers: *loggers
+
+      jobProxyLoggers:
+        - name: debug
+          compression: zstd
+          minLogLevel: debug
+          writerType: file
+          useTimestampSuffix: true
+          rotationPolicy: &rotationPolicyJobs
+            maxTotalSizeToKeep: 104857600  # 100Mi
+            rotationPeriodMilliseconds: 900000  # 15Min
+          categoriesFilter:
+            type: exclude
+            values: [ "Bus", "Concurrency" ]
+        - name: info
+          minLogLevel: info
+          writerType: file
+          rotationPolicy: *rotationPolicyJobs
+        - name: error
+          minLogLevel: error
+          writerType: stderr
+
+      resources:
+        # Allocate resources for exec node container
+        requests:
+          cpu: 1
+          memory: 1Gi
+        limits:
+          cpu: 10
+          memory: 10Gi
+
+      locations:
+        - locationType: Logs
+          path: /yt/exec-node-logs
+        - locationType: ChunkCache
+          path: /yt/node-data/chunk-cache
+        - locationType: Slots
+          path: /yt/node-data/slots
+        - locationType: ImageCache
+          path: /yt/node-data/image-cache
+
+      volumeMounts:
+        - name: exec-node-logs
+          mountPath: /yt/exec-node-logs
+        - name: node-data
+          mountPath: /yt/node-data
+          # mountPropagation: Bidirectional  # Enable for tmpfs in jobs
+
+      volumeClaimTemplates:
+        - metadata:
+            name: node-data
+          spec:
+            accessModes: [ "ReadWriteOnce" ]
+            resources:
+              requests:
+                storage: 20Gi
+        - metadata:
+            name: exec-node-logs
+          spec: *logsVolumeSpec
+
+      # privileged: true  # Enable for tmpfs in jobs
+
+      jobResources:
+        # Allocate resources for jobs container
+        requests:
+          cpu: 4
+          memory: 4Gi
+        limits:
+          cpu: 10
+          memory: 10Gi
+
+      jobEnvironment:
+        # Add CRI containerd sidecar
+        cri:
+          apiRetryTimeoutSeconds: 180
```

Comment:
Do we need to customize it?

Reply:
Right now - yes, some tuning is required.